test/com/facebook/buck/artifact_cache/InMemoryArtifactCache.java

```diff
@@ -52,7 +52,7 @@ public CacheResult fetch(RuleKey ruleKey, LazyPath output) {
     try {
       Files.write(output.get(), artifact.data);
     } catch (IOException e) {
-      Throwables.throwIfUnchecked(e);
```

Comment:
I'm confused by the existing code: it only catches `IOException`, but then only rethrows unchecked exceptions which `IOException` is not.

It seems it's a mistake in the guava 20 update: it should have added a `throw new RuntimeException(e)` here.

I see that's what the new code does, so that's all right, ignore me.